mne/preprocessing/infomax_.py

```diff
@@ -292,7 +292,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
             # for extended Infomax
             if extended:
                 signs = np.identity(n_features)
-                signs.flat[slice(0, n_features * n_subgauss, n_features)]
+                # XXX ?
+                # signs.flat[slice(0, n_features * n_subgauss, n_features)]
```

Comment:
Basically the question is, how do I make the weights blow up in ICA? We can create arbitrary examples using `RawArray` and `EpochsArray` so hopefully it's not too bad.